pulsar-broker-common/src/main/java/com/yahoo/pulsar/broker/ServiceConfiguration.java

```diff
@@ -179,6 +179,11 @@
     private int managedLedgerCursorMaxEntriesPerLedger = 50000;
     // Max time before triggering a rollover on a cursor ledger
     private int managedLedgerCursorRolloverTimeInSeconds = 14400;
+    // Max number of entries to append to a ledger before triggering a rollover
+    // A ledger rollover is triggered on these conditions Either the max
+    // rollover time has been reached or max entries have been written to the
+    // ledged and at least min-time has passed
+    private int managedLedgerMaxUnackedRangesToPersist = 1000;
```

Comment:
I think we don't [persistPosition](https://github.com/merlimat/pulsar/blob/ecf450173d170a9324425c94f0480b8d3feb8c90/managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java#L1914) only on `ledger rollover` but persist `markDeletePosition` based on `managedLedgerDefaultMarkDeleteRateLimit`